src/Microsoft.AspNetCore.TestHost/TestServer.cs

```diff
@@ -27,6 +27,19 @@ public TestServer(IWebHostBuilder builder)
             _hostInstance = host;
         }
 
+        public TestServer(IWebHostBuilder builder, IFeatureCollection featureCollection)
```

Comment:
It should be assignable in the constructor, especially if you don't initialize it otherwise. Try removing the `IServer.` prefix from the Features property to see if that's confusing it.